sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/MethodsShouldUseBaseTypes.cs

```diff
@@ -13,8 +13,8 @@ public class B : A
     {
         public void Method() { }
 
-        public void Test_01(B foo) // Noncompliant {{Consider using more general type 'Test_01.A' instead of 'Test_01.B'.}}
-//                            ^^^
+        public void Test_01(B foo) // Noncompliant {{Consider using more general type 'Test_01.A' instead of 'Test_01.B'.}}

```

Comment:
What changed in this line (and test 16)?

Reply:
Nothing...